tests/test_viewport.py

```diff
@@ -53,3 +55,27 @@ async def test_set_viewport(websocket, context_id):
         "type": "number",
         "value": 300
     }]] == result["result"]["value"]
+
+
+@pytest.mark.asyncio
+async def test_set_viewport_unsupported(websocket, context_id):
+    await goto_url(websocket, context_id, "about:blank")
+
+
+    with pytest.raises(Exception) as exception_info:
+      await execute_command(
+          websocket, {
+              "method": "browsingContext.setViewport",
+              "params": {
+                  "context": context_id,
+                  "viewport": {
+                      "width": 10000001,
```

Comment:
Can we have a link to the code that the max value is set. Atm the number seems to come from thin air.

Reply:
There is a link to the code where the error is thrown.

Reply:
Updated